### PR TITLE
Update the Galaxy Code of Conduct

### PR DESCRIPTION
**Pull request by beatrizserrano**
This PR updates the content of the CoC based on the discussions in the google doc. I made some extra adjustments:

- Included a sentence at the beginning stating what the CoC is (thanks Gareth for the suggestion).
- Reporting
  - Creation of a form for (anonymous) reporting. Inspiration taken from the Bioconductor community.
  - Creation of a mailing list for the members of the committee and added it as an option to report.
- Committee
  - Included the members of the CoC in a table with emails embedded. Please fix the content as needed.  
  - After meeting the Bioconductor CoC committee, added Laurent Gatto as ombudsperson and a couple of sentences explaining when to contact the Ombudsperson.
- Moved the historical note to the end to give some context to this change.

### CODE REVIEW

File: content/community/coc/index.md [hunk 1]
```diff
@@ -5,121 +5,99 @@ title: "Galaxy Project Code of Conduct"
...
+The Galaxy Code of Conduct Committee has appointed an Ombudsperson from the Bioconductor community to provide independent and impartial oversight and to resolve conflicts. This person can be contacted in case you believe that other avenues of reporting may be biased.
+
+| Member | Pronouns | Language | Location | Relevant information |
+| --- | --- | --- | --- | --- | --- |
+| [Laurent Gatto](mailto:laurent.gatto@gmail.com) | he/him | FR, EN | BE | Ombudsperson, member of the Bioconductor community |
 
 ### Attribution & Acknowledgements
 
-This code of conduct is based on the [Open Code of Conduct from the TODOGroup](https://todogroup.org/blog/followup-open-code-of-conduct/).
+This CoC is based on the Geek Feminism Code of Conduct and the [Open Code of Conduct from the TODOGroup](https://todogroup.org/blog/followup-open-code-of-conduct/).
```
Comment (assuntad23) [inline content/community/coc/index.md:93]: should we link to their CoC?

**Review by frederikcoppens — COMMENTED** [review]
NL can be added as language for me, and I guess also FR (although not sure about the French vocabulary for these kinds of topics).

**Comment by beatrizserrano** [comment]
> NL can be added as language for me, and I guess also FR (although not sure about the French vocabulary for these kinds of topics).

We can put them in order. Would it be: NL, EN, FR?

**Review by hexylena — APPROVED** [review]

File: content/community/coc/index.md [hunk 2]
```diff
@@ -5,121 +5,99 @@ title: "Galaxy Project Code of Conduct"
...
+- ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’.
+- Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”
+- Communicating in a ‘tone’ you don’t find congenial.
+- Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions.
+
+
+### Reporting
+
+If you are being harassed by a member of the Galaxy Community, notice that someone else is being harassed, or have any other concerns, please send a report via [this form](https://forms.gle/gEMubTrW9ifwemqe6) that will be forwarded to the [Galaxy CoC Committee](#the-galaxy-coc-committee). Alternatively, you can [email the Code of Conduct Committee](mailto:code-of-conduct@lists.galaxyproject.org). If you are uncomfortable reporting to the Code of Conduct committee as a group, you can contact any individual committee member via email. If the person who is harassing you is on the team, they will recuse themselves from handling your incident. We will respond as promptly as we can.
```
Comment (mvdbeek) [inline content/community/coc/index.md:59]: Should we put the form url under a url-service ? e.g `gxy.io/coc` ?

File: content/community/coc/index.md [hunk 3]
```diff
@@ -5,121 +5,99 @@ title: "Galaxy Project Code of Conduct"
...
+
+To protect volunteers from abuse and burnout, we reserve the right to reject any report we believe to have been made in bad faith.
```
Comment (mvdbeek) [inline content/community/coc/index.md:61]: Do we need to specify what good/bad faith consititutes ?

File: content/community/coc/index.md (hunk 2 above)
Comment (hexylena) [reply content/community/coc/index.md:59]: That'd be super helpful!

**Review by mvdbeek — APPROVED** [review]
Very nice, thanks for the work!

File: content/community/coc/index.md (hunk 3 above)
Comment (beatrizserrano) [reply content/community/coc/index.md:61]: That's a good point. And a hard question, I think it's hard to define. We would need to discuss this in the first committee meeting.

File: content/community/coc/index.md (hunk 1 above)
Comment (beatrizserrano) [reply content/community/coc/index.md]: done! :)

**Review by annefou — APPROVED** [review]
Excellent! This should be merge as soon as possible (because we care about it, isn't it?).

File: content/community/coc/index.md (hunk 2 above)
Comment (hexylena) [reply content/community/coc/index.md]: https://github.com/galaxyproject/infrastructure-playbook/pull/38

**Pull request merged**